index.html

```diff
@@ -180,40 +180,44 @@ <h2>Contact me</h2>
               <input
                 type="text"
                 name="name"
-                placeholder="Aimable"
+                id="name"
+                placeholder="Name"
                 maxlength="30"
                 required
               />
             </li>
             <li class="li1-li_st">
               <input
                 type="email"
-                name="email"
-                placeholder="bancunguye@gmail.com"
+                name="Email"
+                id="email"
+                pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$"
+                placeholder="Email"
                 required
               />
             </li>
             <li class="li2-li_st">
               <textarea
                 name="comments"
-                placeholder="comment"
+                placeholder="Write a message"
                 maxlength="500"
+                id="textarea"
                 required
               ></textarea>
             </li>
-            <li class="li3-li_st">
-              <input
-                class="btn-btn-default"
-                type="submit"
-                name="submit"
-                value="Get in touch"
-              />
-            </li>
           </ul>
+          <div class="contact_button">
+            <small id="alertmessage"></small>
```

Comment:
- [ ] [**OPTIONAL**]Kindly make sure the design below stick to the Figma design :smile: 
![Screenshot from 2023-03-10 19-39-36](https://user-images.githubusercontent.com/80458199/224385984-55ba2da3-6d84-4d90-aa2c-0771bec8d2c5.png)